packages/datatrak-web/src/__tests__/features/Questions/AutocompleteQuestion.test.tsx

```diff
@@ -66,21 +66,21 @@ describe('Autocomplete Question', () => {
     renderComponent(<AutocompleteQuestion {...props} />);
   });
 
-  it('renders all the options when there is no attribute filtering involved', () => {
+  it('renders all the options when there is no attribute filtering involved', async () => {
     renderComponent(<AutocompleteQuestion {...props} />);
 
     const openButton = screen.getByTitle('Open');
     openButton.click();
 
-    const displayOptions = screen.getAllByRole('option');
+    const displayOptions = await screen.findAllByRole('option');
```

Comment:
I was getting inconsistent results on the test where sometimes it was loading results and sometimes not so adding this await seems to make it consistent.